lib/src/model/movies.dart

```diff
@@ -1,15 +1,9 @@
 import 'movie_result.dart';
 
 class Movies {
-  late final int page;
-  late final int totalResults;
-  late final int totalPages;
```

Comment:
why did you removed this variables? you should map the entire response, even if you don't use all the elements in iot

Reply:
re-added